tools/build/docker/install-everything.sh

```diff
@@ -4,16 +4,26 @@
 apk update
 apk add perl perl-io-socket-ssl perl-dev redis libarchive-dev libbz2 openssl-dev zlib-dev
 apk add imagemagick imagemagick-perlmagick libwebp-tools libheif
-apk add g++ make pkgconf gnupg wget curl nodejs nodejs-npm
+apk add g++ make pkgconf gnupg wget curl nodejs nodejs-npm file
 apk add shadow s6 s6-portable-utils
 
 #Hey it's cpanm
 curl -L https://cpanmin.us | perl - App::cpanminus
 
+#!AVX
+if [ $(uname -m) == 'x86_64' ]; then
+  cpanm --notest --installdeps Alien::FFI
+  curl -L -s https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/Alien-FFI-0.25.tar.gz | tar -xz
+  cd Alien-FFI-0.25
+  sed -i 's/--disable-builddir/--disable-builddir --with-gcc-arch=x86-64-v2/' alienfile
```

Comment:
I recall seeing that Alien:FFI builds libffi with AVX support by default(according to the related GH issues), is the gcc arch override really necessary?

Reply:
(that or it needs to be set in order to override alpine's built-in libffi? This is another reason more comments'd be awesome 👍)

Reply:
That I'm not sure. Alpine's libffi build has AVX so I assumed that GCC defaults to it enabled. It's a *just in case* thing.

I'm gonna try on a host that has AVX, if simply installing Alien::FFI is enough then it could be simplified to an one-liner.